### ISSUE TITLE
Traceback when double-clicking list item

### ISSUE THREAD

**Issue by ryanbugden** [robofont-mechanic/mechanic-2#46]
Problem: 
Double-clicking a list item on the Releases pop-up for extensions like Batch or Space Ranger leads to a traceback and no action.

Here is the traceback:
```
Traceback (most recent call last):
  File "lib/doodleDelegate.pyc", line 90, in sendEvent_
  File "/Applications/RoboFont-4_5b2.app/Contents/Resources/lib/python3.12/vanilla/vanillaBase.py", line 539, in action_
    self.callback(sender)
  File "/Users/Ryan/Library/Application Support/RoboFont/plugins/Mechanic2.roboFontExt/lib/mechanic2/ui/controller.py", line 90, in releasesDoubleClickCallback
    self.item.openURL(url=releaseItem["html_url"], background=True)
    ^^^^^^^^^^^^^^^^^
AttributeError: 'ExtensionRepositoryItem' object has no attribute 'openURL'. Did you mean: 'openUrl'?
```

I think this line leads to the issue:
https://github.com/robofont-mechanic/mechanic-2/blob/04ecf0bd3f7e7762eaaf1c2276b8f9c89945f865/Mechanic2.roboFontExt/lib/mechanic2/ui/controller.py#L90

Discussion: 
https://discord.com/channels/1052516637489766411/1260673493142601809

**Comment by ryanbugden** [comment]
The traceback was helpful! Looks like it's just a typo. 

Should be `self.item.openUrl(url=releaseItem["html_url"], background=True)`

https://github.com/robofont-mechanic/mechanic-2/blob/04ecf0bd3f7e7762eaaf1c2276b8f9c89945f865/Mechanic2.roboFontExt/lib/mechanic2/extensionItem.py#L327

**Comment by typemytype** [comment]
should be fixed in 421b0b0
